sql-to-dbsp-compiler/lib/sqllib/src/lib.rs

```diff
@@ -36,13 +34,56 @@ use dbsp::{
     utils::*,
     DBData, OrdIndexedZSet, OrdZSet, OutputHandle, SetHandle, ZSetHandle, ZWeight,
 };
-use num::{Signed, ToPrimitive};
+use num::{PrimInt, Signed, ToPrimitive};
+use num_traits::{Pow, Zero};
 use rust_decimal::{Decimal, MathematicalOps};
 use std::fmt::Debug;
 use std::marker::PhantomData;
 use std::ops::{Add, Deref, Neg};
 use std::str::FromStr;
 
+pub trait ToInteger<T>
```

Comment:
Can you add comments to these traits explaining what they are for (it's obvious what they do from the signature, but the intention needs to be clarified)?